src/raftstore/store/peer_storage.rs

```diff
@@ -815,14 +822,21 @@ impl PeerStorage {
         Ok(last_index)
     }
 
-    pub fn compact_to(&mut self, idx: u64) {
-        self.cache.compact_to(idx);
+    #[inline]
```

Comment:
Do we need to inline the function manually?

Reply:
Maybe not. Nowadays, compiler could optimize it.

Reply:
I think inline it explicitly is never bad if we indeed need it to be inline.